scripts/download_synthea.sh

```diff
@@ -1,14 +1,26 @@
 #!/bin/bash
+# Download Synthea Data in the correct directory PyVertical/data/synthea
+cd ../data/
+GITKEEP_exists=/synthea/.gitkeep
+if test -f "$GITKEEP_exists"; then
+    rm /synthea/.gitkeep
+fi
+
+git clone https://github.com/synthetichealth/synthea.git
```

Comment:
I've currently got synthea added as a submodule, so we shouldn't need to clone it, should we?

Perhaps, to keep `PyVertical` clean, we should remove synthea as a submodule and instead just provide instructions on *what* script to run to generate the data and *where* to place the data?

So replace this script with instructions, and remove the synthea submodule. Do you think that's a good idea?

Reply:
Yes, makes sense. I have modified the script with the instructions instead of cloning synthea repo.